### ISSUE TITLE
captive portal no redirection.

### ISSUE THREAD

**Issue by jepunband** [s0lst1c3/eaphammer#62]
Hi , so i just downloaded eaphammer and tried the captive portal attack with this method:

./eaphammer --bssid 1C:7E:E5:97:79:B1 --essid HappyMealz --channel 149 --interface wlan0 --captive-portal

AP starts, client gets connected but it does not redirect to the captive portal. Localhost can view the portal and apache is started.

Client should see signin redirection to portal once connected to wireless AP.

Need some help. Thanks.

**Comment by shinhub** [comment]
I do not think EAPHammer captive-portal is supposed to handle requests and redirect. You have to deal with this.
A little reminder on how captive-portal redirections usually work might be needed: your device will try to reach predefined Internet pages and look for expected content to check if Internet is reachable. If a page is reachable, but does not contain expected content, then it should be a captive portal, and devices often show a dedicated browser to display the content that is triggered.
In order to do that, as EAPHammer just enable Apache web server when you run captive-portal scenario, you must config your Apache VirtualHost to redirect any request (except ones to the pages you actually use as captive portal) to you captive portal index.
I did as so for instance:
```
RewriteEngine on
	RewriteRule "!(captive\.html|\.js|\.css)$" "captive.html" [R]
```

**Comment by s0lst1c3** [comment]
Resolved by #150
